waiter/src/waiter/state.clj

```diff
@@ -539,7 +539,11 @@
    initial-state]
   (when (some nil? (vals initial-state))
     (throw (ex-info "Initial state contains nil values!" initial-state)))
-  (let [max-backoff-exponent (int (max 1 (inc (/ (Math/log max-blacklist-time-ms) (Math/log blacklist-backoff-base-time-ms)))))
+  (let [max-backoff-exponent (->> (/ (Math/log max-blacklist-time-ms) (Math/log blacklist-backoff-base-time-ms))
+                                  Math/exp
```

Comment:
It doesn't look like this was in the original code, was it a bug that it was missing?

Reply:
Yes, this was a bug that got fixed.

Reply:
👍